fendermint/fendermint/eth/api/src/apis/eth.rs

```diff
@@ -731,14 +731,19 @@ where
     let params = RawBytes::serialize(params).context("failed to serialize position to IPLD")?;
     let height = data.query_height(block_id).await?;
 
-    let ret = data
+    let Ok(ret) = data
         .read_evm_actor::<evm::GetStorageAtReturn>(
             address,
             evm::Method::GetStorageAt,
             params,
             height,
         )
-        .await?;
+        .await
+    else {
+        // TODO Actually compare the actor type and return empty if not an EVM actor.
+        //  See https://github.com/consensus-shipyard/ipc-monorepo/issues/506.
+        return Ok(Default::default());
```

Comment:
With the placholder actor, where `None` is returned by the query, the existing code returns it as `"00...00"`. To be consistent with that, this shouldn't return `""` either.

Reply:
True, good catch!